_Week5/group4/README.md

```diff
@@ -0,0 +1,16 @@
+# Week 5 Homework Assignment, Group 4: `graphsum`
```

Comment:
It would be helpful if you told the user what he/she must change to run the program, namely the file path.

Reply:
I added a setup section that I hope clears things up.